packages/form-core/src/FormApi.ts

```diff
@@ -160,6 +160,20 @@ export interface FormOptions<
     formApi: FormApi<TFormData, TFormValidator>
   }) => void
   transform?: FormTransform<TFormData, TFormValidator>
+  /**
+   * When true, all previous validation errors will be cleared when running validation.
+   *
+   * While `true`:
+   *  - `onMount` and `onBlur` errors will be cleared when `onChange` triggers validation.
+   *
+   * While `false`:
+   *  - An `onMount` error will never be cleared.
+   *  - An `onBlur` error will only be cleared on the next `onBlur` event.
+   *  - An `onChange` error will only be cleared on the next `onChange` event.
+   *
+   * @default false
+   */
+  automaticallyResetValidators?: boolean
```

Comment:
The action this flag enables is that errors are removed when a new validator is run, I think we should better explain this.

I like `resetOnRevalidate` to indicate that an action (reset) is performed when something happens (onRevalidate) or `resetErrorsOnRevalidate` if we want to be extra explicit.

What do you think?